src/main/scala/io/circe/yaml/Parser.scala

```diff
@@ -129,5 +138,24 @@ final case class Parser(
 }
 
 object Parser {
-  val default: Parser = Parser()
+  final case class Default(
+    allowDuplicateKeys: Boolean = false,
+    allowRecursiveKeys: Boolean = false,
+    bufferSize: Int = 1024,
+    label: String = "reader",
+    maxAliasesForCollections: Int = 50,
+    parseComments: Boolean = false,
+    useMarks: Boolean = true
+  ) extends Parser({
+        val builder = LoadSettings.builder
+        builder.setAllowDuplicateKeys(allowDuplicateKeys)
+        builder.setAllowRecursiveKeys(allowRecursiveKeys)
+        builder.setBufferSize(bufferSize)
+        builder.setLabel(label)
+        builder.setMaxAliasesForCollections(maxAliasesForCollections)
+        builder.setParseComments(parseComments)
+        builder.setUseMarks(useMarks)
+        builder.build
```

Comment:
This builder uses a fluent pattern (each of these "set..." calls returns `this`) - meaning there's no need for this block with `val builder = ...` and a bunch of side-effecting statments. This could be written e.g.
```scala
extends Parser(
  LoadSettings.builder
    .setFoo(...)
    .setBar(...)
    .build
  )
  ```

Reply:
Good point, I've simplified the code, thanks.